saleor/tests/e2e/orders/test_delete_draft_order.py

```diff
@@ -0,0 +1,76 @@
+import pytest
+
+from .. import DEFAULT_ADDRESS
+from ..product.utils.preparing_product import prepare_product
+from ..shop.utils.preparing_shop import prepare_shop
+from ..utils import assign_permissions
+from .utils.draft_order_create import draft_order_create
+from .utils.draft_order_delete import draft_order_delete
+from .utils.draft_order_update import draft_order_update
+from .utils.order_lines_create import order_lines_create
+from .utils.order_query import order_query
+
+
+@pytest.mark.e2e
+def test_delete_draft_order_CORE_0205(
+    e2e_staff_api_client,
+    permission_manage_products,
+    permission_manage_channels,
+    permission_manage_product_types_and_attributes,
+    permission_manage_shipping,
+    permission_manage_orders,
+):
+    # Before
+    permissions = [
+        permission_manage_products,
+        permission_manage_channels,
+        permission_manage_shipping,
+        permission_manage_product_types_and_attributes,
+        permission_manage_orders,
+    ]
+    assign_permissions(e2e_staff_api_client, permissions)
+
+    price = 10
+
+    (
+        result_warehouse_id,
+        result_channel_id,
+        _,
+        result_shipping_method_id,
+    ) = prepare_shop(e2e_staff_api_client)
+
+    _, result_product_variant_id, _ = prepare_product(
+        e2e_staff_api_client, result_warehouse_id, result_channel_id, price
+    )
+
+    # Step 1 - Create draft order
+    draft_order_input = {
+        "channelId": result_channel_id,
+        "userEmail": "test_user@test.com",
+        "shippingAddress": DEFAULT_ADDRESS,
+        "billingAddress": DEFAULT_ADDRESS,
+    }
+    data = draft_order_create(
+        e2e_staff_api_client,
+        draft_order_input,
+    )
+    order_id = data["order"]["id"]
+    assert order_id is not None
+
+    # Step 2 - Add lines to the order
+    lines = [{"variantId": result_product_variant_id, "quantity": 1}]
+    order_lines = order_lines_create(e2e_staff_api_client, order_id, lines)
+    order_product_variant_id = order_lines["order"]["lines"][0]["variant"]["id"]
+    assert order_product_variant_id == result_product_variant_id
+
+    # Step 3 - Update order's shipping method
+    input = {"shippingMethod": result_shipping_method_id}
+    draft_order = draft_order_update(e2e_staff_api_client, order_id, input)
+    order_shipping_id = draft_order["order"]["deliveryMethod"]["id"]
+    assert order_shipping_id is not None
+
+    # Step 4 - Cancel the order
```

Comment:
Can you update the description to `Delete the order` cancel is related to completed orders which cannot be deleted